docroot/modules/custom/mass_content/mass_content.module

```diff
@@ -639,7 +639,7 @@ function mass_content_form_taxonomy_term_collections_form_alter(array &$form, Fo
     $form['field_sort_order']['#access'] = FALSE;
     $form['field_primary_parent']['#access'] = FALSE;
     $form['field_landing_page']['#access'] = FALSE;
-    $form['field_collection_logo']['#access'] = FALSE;
+    $form['field_reusable_label']['#access'] = FALSE;
```

Comment:
This is fine for now. Next time I think its better to remove from the form display.

Reply:
I think the code conditionally removes the fields, which is hard to achieve from the UI